packages/loader/container-loader/src/container.ts

```diff
@@ -1682,7 +1682,9 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         // Both protocol and context should not be undefined if we got so far.
 
         if (this._context?.disposed === false) {
-            this.context.setConnectionState(state, this.clientId);
+            this.context.setConnectionState(
```

Comment:
I'd add big comment on why we are doing it. And I'd probably even go further and open a task to rename setConnectionState() to something that more clearly describes purpose of this API - "it's Ok to send ops"

Reply:
@vladsud I agree about changing the name to be more indicative of what this controls, but wouldn't that make it a breaking change? Could be a follow up item

Reply:
Yes, follow up is fine.
I'd do it in two steps:
1. Introduces new names at container runtime layer (keeping old one around). Loader layer uses new name if such property exists or uses old name if it does not.
2. After some time get rid of old names.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/1821